.github/scripts/install_nvidia_utils_linux.sh

```diff
@@ -3,7 +3,7 @@
 set -eou pipefail
 
 DISTRIBUTION=$(. /etc/os-release;echo $ID$VERSION_ID) \
-DRIVER_FN="NVIDIA-Linux-x86_64-510.60.02.run"
+DRIVER_FN="NVIDIA-Linux-x86_64-515.57.run"
```

Comment:
Why are we updating driver here? Aren't forward compatibility guarantee for CUDA-11.x applies for 11.7?